src/backend/serial/u64/scalar.rs

```diff
@@ -193,53 +193,47 @@ impl Scalar52 {
 
     /// Compute `a * b`
     #[inline(always)]
-    pub (crate) fn mul_internal(a: &Scalar52, b: &Scalar52) -> [u128; 9] {
+    pub(crate) fn mul_internal(a: &Scalar52, b: &Scalar52) -> [u128; 9] {
         let mut z = [0u128; 9];
 
-        z[0] = m(a[0],b[0]);
-        z[1] = m(a[0],b[1]) + m(a[1],b[0]);
-        z[2] = m(a[0],b[2]) + m(a[1],b[1]) + m(a[2],b[0]);
-        z[3] = m(a[0],b[3]) + m(a[1],b[2]) + m(a[2],b[1]) + m(a[3],b[0]);
-        z[4] = m(a[0],b[4]) + m(a[1],b[3]) + m(a[2],b[2]) + m(a[3],b[1]) + m(a[4],b[0]);
-        z[5] =                m(a[1],b[4]) + m(a[2],b[3]) + m(a[3],b[2]) + m(a[4],b[1]);
-        z[6] =                               m(a[2],b[4]) + m(a[3],b[3]) + m(a[4],b[2]);
-        z[7] =                                              m(a[3],b[4]) + m(a[4],b[3]);
-        z[8] =                                                             m(a[4],b[4]);
+        z[0] = m(a[0], b[0]);
+        z[1] = m(a[0], b[1]) + m(a[1], b[0]);
+        z[2] = m(a[0], b[2]) + m(a[1], b[1]) + m(a[2], b[0]);
+        z[3] = m(a[0], b[3]) + m(a[1], b[2]) + m(a[2], b[1]) + m(a[3], b[0]);
+        z[4] = m(a[0], b[4]) + m(a[1], b[3]) + m(a[2], b[2]) + m(a[3], b[1]) + m(a[4], b[0]);
+        z[5] = m(a[1], b[4]) + m(a[2], b[3]) + m(a[3], b[2]) + m(a[4], b[1]);
+        z[6] = m(a[2], b[4]) + m(a[3], b[3]) + m(a[4], b[2]);
+        z[7] = m(a[3], b[4]) + m(a[4], b[3]);
+        z[8] = m(a[4], b[4]);
```

Comment:
Here is a location where the whitespace assists in understanding how the code works.

The "stair-stepping" approach shows how the indices line up, which is absent from the rustfmt'd version (and looks quite a bit like the original `ref10` C implementation.

This is a detail I think it would be a pity to lose.